src/modules/pos/validator/components/StakeStatus/statusMessages.js

```diff
@@ -64,12 +64,12 @@ const getSuccessMessage = (t, locked, unlockable, selfUnvote = { confirmed: 0 })
   return '';
 };
 
-const voteStatusMessages = (t, statusInfo) => ({
+const stakeStatusMessages = (t, statusInfo) => ({
   ...statusMessages(t),
   [txStatusTypes.broadcastSuccess]: {
-    title: t('Votes are submitted'),
+    title: t('Stakes are submitted'),
```

Comment:
Should it be `Tokens are staked` ? You submit your vote, but do we say the same way about stakes?